public/app/plugins/datasource/loki/querybuilder/components/LokiQueryBuilderOptions.tsx

```diff
@@ -243,4 +278,20 @@ function getCollapsedInfo(
   return items;
 }
 
+function getDefaultQueryDirection(app?: CoreApp) {
+  if (app !== CoreApp.Explore) {
+    /**
+     * The default direction is forward because the default sort order is Descending.
```

Comment:
This comment doesn't seem to line up with the return?

Reply:
Ah yes, changed the return but not the comment.